src/mnode/mnode-pastel.cpp

```diff
@@ -1428,6 +1532,26 @@ std::unique_ptr<CPastelTicket> CArtTradeTicket::FindArtRegTicket() const
     return std::move(chain.front());
 }
 
+void CArtTradeTicket::SetArtRegTicketTxid(const std::string& _NftRegTxid)
+{
+   nftRegTnxId = std::move(_NftRegTxid);
```

Comment:
calling std::move on a const object is not a good practice

Reply:
Ok, done.